app/views/admin/bikes/edit.html.haml

```diff
@@ -20,7 +20,7 @@
         - if @bike.current_stolen_record.present?
           = @bike.dl_list_item(@bike.current_stolen_record.phone, 'Phone')
       %dl.dl-horizontal
-        = @bike.dl_list_item(@bike.created_at.strftime('%m-%d-%Y'), 'Created')
+        = @bike.dl_list_item(@bike.created_at.strftime('%Y-%m-%d'), 'Created')
```

Comment:
I was going to ask you about this when I noticed the file path and realized what those pages are for (d'oh!). I had already made some commits by then, but now I know. I'll back them out before the final version